Additional release notes for Keycloak 24 (#27339)
keycloak/keycloak: docs/documentation/server_admin/topics/clients/client-policies.adoc

    Applies for clients with the client role of the specified name. Typically you can create a client role of specified name to requested clients and use it as a "marker role" to make
    sure that specified client policy will be applied for requested clients.

> NOTE: There is often a use-case for require that particular client policy is applied for the specified clients like `my-client-1` and `my-client-2`. The best way to achieve this
> is to use *Client Role* condition in your policy and then create client role of specified name to requested clients. This client role can be used as a "marker role" used solely
> for marking that particular client policy should be used for particular clients.

Reviewer: ```suggestion
NOTE: A use-case often exists for requiring the application of a particular client policy for the specified clients such as `my-client-1` and `my-client-2`. The best way to achieve this result 
is to use a *Client Role* condition in your policy and then a create client role of specified name to requested clients. This client role can be used as a "marker role" used solely
for marking that particular client policy should be used for particular clients.
```
